internal/provider/providers/nowdns/provider.go

```diff
@@ -0,0 +1,169 @@
+package nowdns
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/netip"
+	"net/url"
+	"strings"
+
+	"github.com/qdm12/ddns-updater/internal/models"
+	"github.com/qdm12/ddns-updater/internal/provider/constants"
+	"github.com/qdm12/ddns-updater/internal/provider/errors"
+	"github.com/qdm12/ddns-updater/internal/provider/headers"
+	"github.com/qdm12/ddns-updater/internal/provider/utils"
+	"github.com/qdm12/ddns-updater/pkg/publicip/ipversion"
+)
+
+type Provider struct {
+	domain        string
+	host          string
```

Comment:
you can remove the host field if it can only ever be `"@"`

Reply:
Removed.

Reply:
It has to be passed as "@", otherwise the API throws a "nohost" error.